topbeat/tests/system/config/topbeat.yml.j2

```diff
@@ -13,10 +13,11 @@ topbeat.procs: [
 
 # Statistics to collect (all enabled by default)
 topbeat.stats:
-  system: {{ "false" if system_stats == false else "true" }}
```

Comment:
Looks like this is 4 spaces now. Does it still work? Is there a need to change this?